### ISSUE TITLE
new vehicles endpoint

### ISSUE THREAD

**Issue by novapulsar** [route4me/route4me-nodejs-sdk#29]
/api.v4/vehicles.php

**Comment by maxkoryukov** [comment]
I have fixed the path, but seems, like the API want response for `GET` queries:

https://travis-ci.org/route4me/route4me-nodejs-sdk/builds/196479289  (if you will change something on the API server - just **restart** this build.

I have tested API access with curl, the same result - **404, Not Found**.

**Comment by maxkoryukov** [comment]
@novapulsar I have create a PR for this - #30

**Comment by Olman62** [comment]
/api.v4/vehicles.php doesn't exist.

There is endpoint /api.v4/vehicle.php, but it's not workable yet.

**Comment by novapulsar** [comment]
get works

**Comment by h2rd** [comment]
hi, we don’t have /api.v4/vehicles.php but we have /api.v4/vehicle.php

**Comment by maxkoryukov** [comment]
@h2rd , yep, `/vehicle.php` works better))

But new endpoint returns empty response:

```
$ curl -G -X GET https://www.route4me.com/api.v4/vehicle.php -d api_key=11111111111111111111111111111111 -v
> 
```

But it is not a problem though. I could wait;) It's because just one step remains - press **Merge PR** for #30. When this PR **will pass all tests** - I will merge it. 

OR: I could rewrite my tests (the tests for `vehicle.php` is the deepest tests in the entire fixture), and switch INTEGRATION tests from `vehicle` to `avoidance zone`

**Comment by maxkoryukov** [comment]
@h2rd, how to add test data (several vehicles) to public API for testing apikey?

**Comment by h2rd** [comment]
hi @maxkoryukov, i'll give you response tomorrow, i need check that endpoint and i'll create / update wiki.

**Comment by maxkoryukov** [comment]
@h2rd, I just switched to `optimizations` for testing (instead of `vehicle`). So currently this issue is not actual;) I won't  rush you)

for your information: fix test cases was done in #36

**Comment by maxkoryukov** [comment]
seems not actual anymore.

* [x] use actual endpoint in SDK
* [x] fix tests
